src/Symfony/Component/HttpFoundation/Session/Storage/Handler/MemcachedSessionHandler.php

```diff
@@ -65,6 +65,8 @@ public function __construct(\Memcached $memcached, array $options = array())
      */
     public function open($savePath, $sessionName)
     {
+        $this->sessionName = $sessionName;
```

Comment:
that could be implemented in the parent class. and then just call paren::open. then sessionName does not need to be protected as well.

Reply:
good idea, done